src/client/hud.cpp

```diff
@@ -488,14 +496,30 @@ void Hud::drawHotbar(u16 playeritem) {
 
 void Hud::drawCrosshair()
 {
-	if (use_crosshair_image) {
+	if (can_draw_selectionindicator)
+	{
+		if (use_selectionindicator_image) {
+			video::ITexture *selectionindicator = tsrc->getTexture("selectionindicator.png");
+			v2u32 size  = selectionindicator->getOriginalSize();
+			v2s32 lsize = v2s32(m_displaycenter.X - (size.X / 2),
+					m_displaycenter.Y - (size.Y / 2));
+			driver->draw2DImage(selectionindicator, lsize,
+					core::rect<s32>(0, 0, size.X, size.Y),
+					nullptr, selectionindicator_argb, true);
+		} else {
+			driver->draw2DLine(m_displaycenter - v2s32(9, 9), m_displaycenter + v2s32(9, 9),
+					selectionindicator_argb);
+			driver->draw2DLine(m_displaycenter + v2s32(9, -9), m_displaycenter + v2s32(-9, 9),
```

Comment:
I'd suggest to use `7`, not `9`, here. `sqrt(1/2)` is circa `0.7071`. Otherwise, the `X` will have longer lines than the `+`.

Reply:
I agree, the 'X' is a little unnecessarily large in the screenshot.

Reply:
7 is too small, this is what 8 looks like:
![image](https://user-images.githubusercontent.com/28972859/78454713-a2424480-764e-11ea-9510-739244a2b4a6.png)

Reply:
7 would make the lines approximately 10 long in each direction (as with `+`).
But 8 looks fine in your screenshot.
(Maybe you should give this magic number a name.)
```
sqrt(2 * 7**2) -> 9.899494936611665
sqrt(2 * 8**2) -> 11.313708498984761
```

Reply:
How do you handle names for magic numbers here? A #define?

Reply:
Yes.

Reply:
How's that?

Reply:
@Desour I can review/test one of your PRs in exchange for an answer here if wanted

Reply:
Oh, sorry for not answering.

It should be fine, I think.

Reply:
Alrighty, thanks!